src/main/webapp/app/exam/overview/exercises/quiz/quiz-exam-submission.component.ts

```diff
@@ -32,13 +32,15 @@ import { SubmissionVersion } from 'app/exam/shared/entities/submission-version.m
     selector: 'jhi-quiz-submission-exam',
     templateUrl: './quiz-exam-submission.component.html',
     providers: [{ provide: ExamSubmissionComponent, useExisting: QuizExamSubmissionComponent }],
-    styleUrls: ['./quiz-exam-submission.component.scss'],
+    styleUrls: ['../../../../quiz/overview/participation/quiz-participation.component.scss', './quiz-exam-submission.component.scss'],
```

Comment:
We re-use the styles from `quiz-participation.component.scss` instead of defining them a second time -> should be extracted to shared component in future anyways